docs/api/field_type_reference.md

```diff
@@ -1435,6 +1436,61 @@ ezpublish:
     The option to automatically get user coordinates through the "Locate me" button
     is only available when the back office is served through the `https://` protocol.
 
+## Matrix Field Type
+
+This Field represents and handles a table of rows and columns of data.
+
+| Name     | Internal name | Expected input |
+|----------|---------------|----------------|
+| `Matrix` | `ezmatrix`    | `array`        |
+
+The Matrix Field Type is available via the eZ Platform Matrix Bundle
+provided by the https://github.com/ezsystems/ezplatform-matrix-fieldtype package.
```

Comment:
consider rephrasing:

provided in/by the package accessible from/you can get from <insert the link>

Reply:
or provided by the [ezplatform-matrix-fieldtype](https://github.com/ezsystems/ezplatform-matrix-fieldtype) package.